<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddFond</class>
 <widget class="QDialog" name="AddFond">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>160</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>160</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>160</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>17</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>17</height>
        </size>
       </property>
       <property name="text">
        <string>Специальность</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboSpec"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>17</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>17</height>
        </size>
       </property>
       <property name="text">
        <string>Книга</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboBook"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>17</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>17</height>
        </size>
       </property>
       <property name="text">
        <string>Колличество</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="spinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="butCan">
       <property name="minimumSize">
        <size>
         <width>99</width>
         <height>27</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>99</width>
         <height>27</height>
        </size>
       </property>
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="butAdd">
       <property name="minimumSize">
        <size>
         <width>99</width>
         <height>27</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>99</width>
         <height>27</height>
        </size>
       </property>
       <property name="text">
        <string>Добавить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
